pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -382,7 +382,7 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 			tr("Enable this option to match PCSX2's refresh rate with your current monitor or screen. VSync is automatically disabled when "
 			   "it is not possible (eg. running at non-100% speed)."));
 
-		dialog->registerWidgetHelp(m_ui.integerScaling, tr("Integer Scaling"), tr("Unchecked"),
+		dialog->registerWidgetHelp(m_ui.integerScaling, tr("Integer Upscaling"), tr("Unchecked"),
```

Comment:
In future we should improve what it does, as people dislike the padding even though the tooltip tells what it does.

Reply:
Upscaling is not really correct, scaling was correct as the image is scaled to the nearest perfect divider size for the resolution, upward or downward (whatever fits on the screen).

Reply:
Restoring and unifying then.